### PR TITLE
rm unused buil-dependency

### PR DESCRIPTION
**Pull request by tijlleenders**

### CODE REVIEW

**Comment by moaz-mokhtar** [comment]
This removed dependency related to build scripts `build.rs`, but I see the same behavior. So why do you think this is important?

**Comment by tijlleenders** [comment]
> This removed dependency related to build scripts `build.rs`, but I see the same behavior. So why do you think this is important?

I don't think this crate is being used. That's why I removed it.  
It is not related to `build.rs` - just cleaning up.

File: Cargo.toml [hunk 1]
```diff
@@ -47,7 +47,6 @@ soft = "0.1.1"
 
 # used to generate tests
```
Comment (Mustafa-Eldeeb) [inline Cargo.toml:48]:
```suggestion
```

File: Cargo.toml [hunk 2]
```diff
@@ -47,7 +47,6 @@ soft = "0.1.1"
...
 [build-dependencies]
```
Comment (Mustafa-Eldeeb) [inline Cargo.toml:49]:
```suggestion
```

**Review by Mustafa-Eldeeb — COMMENTED** [review]
You can remove CC from build deps. Also I suggest to clean the whole section

**Review by tijlleenders — COMMENTED** [review]
Ok please merge.

**Review by Mustafa-Eldeeb — APPROVED** [review]

**Pull request merged**